app/controllers/decidim/time_tracker/admin/assignees_controller.rb

```diff
@@ -61,7 +61,7 @@ def update
         def assignees
```

Comment:
```suggestion
        # obtaining the users separately to have them ordered in a nice way
        def assignees
```
We should a comment in this method, otherwise looks weird the verbosity